components/gitpod-db/src/typeorm/user-db-impl.ts

```diff
@@ -123,10 +125,17 @@ export class TypeORMUserDBImpl implements UserDB {
         return user;
     }
 
+    private userCacheKey = "user";
+    private getCacheKey(id: string) {
```

Comment:
Why not make this a unit-level `function getCacheKey`? Saves us the `this.` below :point_down: Or do we want to encourage re-implementing (I assume not due to the `private`).